_episodes/04-faceting-and-filtering.md

```diff
@@ -35,7 +35,7 @@ You can also `invert` the filter to show all records which do not match your sel
 
 >## Let's create a text facet
 >1. Click on the drop down menu at the top of the publisher column and choose `Facet > Text Facet`. The facet will then appear in the left hand panel
->2. Filter by facet by clicking or `include`
+>2. Filter by facet by clicking on each facet or on `include`
 >3. Select multiple values to `include`
```

Comment:
Do you think we should make it clearer here that the difference is:
* To select a single value, just click the relevant line in the facet
* To select multiple values click the "Include" option on the appropriate line in the facet (which only appears when you mouse over the line)

What do you think?

Reply:
Works for me. Anyone else have a suggestion?